### PR TITLE
Multi line code fix

### PR DESCRIPTION
**Pull request by Lucien950**
aight ok it looks ok now.

image

### CODE REVIEW

**Comment by picklepower24** [comment]
Is this mixed with the font fix branch? I see tons of just renamed files, similar to the font fix one.

**Comment by Lucien950** [comment]
> Is this mixed with the font fix branch? I see tons of just renamed files, similar to the font fix one.

oh no the font fix branch was already merged when I branched from dev

**Comment by Lucien950** [comment]
> It's not changing to the code font.
> image

ok this is done
it looks beautiful now!

File: components/VsTextarea.vue [hunk 1]
```diff
@@ -110,32 +113,37 @@ export default class VsTextarea extends Vue {
...
-        highlightText +
+        highLightText +
         afterText +
-        this.contents.substring(endPos!)
-      // This is dumb code don't do this at home.
-      setTimeout(() => {
+        this.contents.substring(endPos)
+    }
+
+    // This is dumb code don't do this at home.
```
Comment (picklepower24) [inline components/VsTextarea.vue:140]: remove troll comment?

**Review by picklepower24 — APPROVED** [review]
It works!!!
image
Just remove that one lil comment that says ur code is dumb! I think it's beautiful :)

**Pull request merged**